BLOG-POST-TEMPLATE.md

```diff
@@ -128,6 +132,35 @@ header for this section is `custom-ids-for-anchor-links`. The text is
 transformed to all lowercase, special characters are ignored, and spaces are
 turned into hyphens.
 
+### Content Summaries
+
+You can define a summary manually, in front matter, or automatically. A manual
+summary takes precedence over a front matter summary, and a front matter summary
+takes precedence over an automatic summary. To manually set a summary for a blog
+post, see https://gohugo.io/content-management/summaries/.
+
+### Internal Links and Shortcodes
+
+Using relative links and shortcodes to refer to other pages is strongly
+encouraged instead of hardcoding paths. Hugo emits an error or warning if a
+document cannot be uniquely resolved. See
+https://gohugo.io/content-management/cross-references/ for how to utilize the
+shortcodes.
+
+For example:
+
+Don't do this:
+
+```md
+[Discontinuation of the GeoLite Legacy Databases](/2018/01/discontinuation-of-the-geolite-legacy-databases/)
+```
+
+Do this:
+
+```md
+[Discontinuation of the GeoLite Legacy Databases]({{< relref "2018/01/discontinuation-of-the-geolite-legacy-databases.md" >}})
+```
+
 ### Images
 
 In markdown, you can embed images with the following format:
```

Comment:
The below should be in backticks. It currently is being rendered. 
![image](https://github.com/user-attachments/assets/9d10e015-2082-41a1-a530-b5d2569a1368)